message.go

```diff
@@ -249,6 +249,10 @@ type MessageEdit struct {
 	Embeds          []*MessageEmbed         `json:"embeds"`
 	AllowedMentions *MessageAllowedMentions `json:"allowed_mentions,omitempty"`
 	Flags           MessageFlags            `json:"flags,omitempty"`
+	// Files to append to the message
+	Files []*File `json:"files,omitempty"`
+	// Overwrite existing attachments
+	Attachments *[]*MessageAttachment `json:"attachments,omitempty"`
```

Comment:
I made this a pointer because an empty array doesn't mean it should be not specified in the json.